pkgs/development/libraries/science/math/hypre/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, mpi
+, gfortran
+}:
+
+stdenv.mkDerivation rec {
+  name = "hypre-${version}";
+  version = "2.15.1";
+
+  src = fetchFromGitHub {
+    owner = "hypre-space";
+    repo = "hypre";
+    rev = "v${version}";
+    sha256 = "1gha571flc6q513137q4416hzpzb6qn2b9hlp3swik5150m91bbb";
+  };
+
+  preConfigure = ''
+    cd src
+    asdfasdfasdfasdf # intentionally fail build
+  '';
+
+  buildInputs = [ mpi gfortran cmake ];
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/hypre-space/hypre;
+    description = "Library for preconditioners and solvers featuring multigrid methods for the solution of large, sparse linear systems of equations";
+    license = licenses.lgpl2;
```

Comment:
```suggestion
    license = licenses.lgpl21;
```

Reply:
Do you have a tool that you use for analyzing licenses? Or do you do this work manually?

Reply:
I usually just look at github repo, it has automatic detection that works most of the times. This time I had to actually look in the README files. So, yes, it is a manual process for me. I think debian might have some tooling to do this automatically.